src/database/index.ts

```diff
@@ -2,8 +2,16 @@ import path from 'path';
 import { TinyPg } from 'tinypg';
 import { migrate as pgMigrate } from 'postgres-migrations';
 
+const HOST: string = process.env.PGHOST ?? 'localhost';
+const USER: string = process.env.PGUSER ?? 'pdc';
+const PASS: string = process.env.PGPASS ?? 'pdc';
+const PORT: string = process.env.PGPORT ?? '5432';
+const DB: string = process.env.PGDATABASE ?? 'pdc';
+
 export const db = new TinyPg({
   root_dir: [path.resolve(__dirname, 'queries')],
+  connection_string:
+    `postgres://${USER}:${PASS}@${HOST}:${PORT}/${DB}`,
```

Comment:
Do we need this?  I believe tinypg automatically picks up all of these env vars since they're standardized within postgresl (that's why migrations worked in the first place)

Reply:
Let me check agin. It may have been one attempt to resolve something but then another thing resolved it.

Reply:
Resolved in f5c08ac